### DISCUSSION TITLE
Music Discord Presence Picture Not Showing?

### DISCUSSION THREAD

**Discussion by Halfs2020** [jojo2357/Music-Discord-Rich-Presence#10] [Q&A]
It's not working?

**Comment by jojo2357** [comment]
Im looking into it, thank you for providing required information. One thing though, is the pictured text file in the `clientdata` folder?

**Reply by jojo2357** [reply]
the pictures only get uploaded to discord, you put the `dat` files (like the one u sent) in that folder. Where is the file that you sent a picture of? That file should go where the image is

**Reply by jojo2357** [reply]
I dont have video taking software, so I will describe better. A `dat` file just contains words like `txt`, not a picture like a `png`. In the folder that you sent a picture of, the one with `chrome.dat` and `demo.dat`, make a new text file, name it `whatever.dat` (remove the default `txt`) and then edit it with notepad (or notepad++). Enter the following:

```
music.ui=groove
id=813535150510964786
<album name>=thelevitatingedition
``` 
Note: replace `<album name>` with a copy paste of the album name
Then save the file and close and reopen the rich presence. Then, play a song from the album.

**Reply by jojo2357** [reply]
> ![image](https://user-images.githubusercontent.com/72892500/108784510-78945380-7567-11eb-95bb-67b510f9cd2a.png)
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/72892500/108784534-834ee880-7567-11eb-98c7-011a25a8dce7.png)
> 
> 
> 
> ![image](https://user-images.githubusercontent.com/72892500/108784546-8ba72380-7567-11eb-9602-95b855e5c8e2.png)
> 
> 
> 
> It's not working?
> 
> 
> 
> 

Remove the < and > from the text file and the png from the folder. What is the name of the file that contains the id and album title?

**Reply by jojo2357** [reply]
What does it say when you hover the big icon?

**Reply by jojo2357** [reply]
And what is the name of the file that you created in client data that has the client id and album name?

**Reply by jojo2357** [reply]
But what folder is the file in? all i have seen are png's in the clientadata folder.

**Reply by jojo2357** [reply]
okay, delete the png as i think it might be causing some issues. If that doesn't work I will try to replicate this issue later and see what is wrong

**Reply by Halfs2020** [reply]
Removed it and it's still doing the same.

**Reply by jojo2357** [reply]
So i took a look at the message history here, and i noticed that one of the pictures you sent was of a rich presence with no images. That means that my program detected the ablum that was playing, but that discord didn't have an image with the key provided. Since the DRP isnt meant to have much of an interface, I'm not sure how to debug it in your case. Could you send me one of the mp3's from that album so i could test it locally? I think that is the best way to solve the issue

**Reply by Halfs2020** [reply]
I can't share these files, sorry.

**Reply by jojo2357** [reply]
Thats unfortunate. This is the first time someone hasn't been unable to upload their own art. Have you tried for other albums?
